src/config-inferrer.spec.ts

```diff
@@ -155,5 +155,27 @@ describe('config inferrer', () => {
                 }
             ]
         })
+    ),
+    it("[.NET] nuget",
```

Comment:
Nit: Local style is lowercase:

```suggestion
    it("[dotnet] nuget",
```